tests/fido_client/FidoClient/request_test.py

```diff
@@ -1,53 +0,0 @@
-import mock
-from mock import patch, Mock
-import pytest
-import six
-
-
-try:
-    from bravado.fido_client import FidoClient
-except ImportError:
-    FidoClient = Mock()
-
-
-@pytest.mark.skipif(six.PY3, reason="twisted doesnt support py3 yet")
-def test_no_timeouts_passed_to_fido():
```

Comment:
Why all the moving of tests? The existing pattern in the lib is one file per method under test. Would like to keep this consistent.